.github/workflows/fakeStore.app/_CodeSignature/CodeResources

```diff
@@ -131,7 +131,7 @@
 		</data>
 		<key>main.jsbundle</key>
 		<data>
-		lJdjPsLiXuiKCW802TFgvJ9wa30=
+		ozrUYoBQ5OeS6gGjvWZiq0+ZR7g=
```

Comment:
should we go back to 40?

Considering the report feels a bit empty, wdyt? 

<img width="456" alt="image" src="https://github.com/bamlab/flashlight/assets/4534323/a9bbfde9-4d79-4863-b138-2e635e62b416">

Reply:
Changed to 35